modules/460-log-shipper/docs/USAGE.md

```diff
@@ -257,3 +254,47 @@ spec:
 ```
 
 > NOTE: If you need logs from only one or from a small group of a pods, try to use the kubernetesPods settings to reduce the number of reading filed. Do not use highly grained filters to read logs from a single pod.
+
+## Collect logs from production namespaces using the namespace label selector option
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: ClusterLoggingConfig
+metadata:
+  name: production-logs
+spec:
+  type: KubernetesPods
+  kubernetesPods:
+    namespaceSelector:
+      labelSelector:
+        matchNames:
+          environment: production
+  destinationRefs:
+  - loki-storage
+```
+
+## Exclude pods or namespaces with a label
+
+There is a preconfigured label to exclude particular namespaces or pods: `log-shipper.deckhouse.io/exclude=true`.
+It can help to stop collecting logs from a namespace or pod without changing global configurations.
```

Comment:
```suggestion
## Exclude Pods or namespaces with a label

```suggestion
There is a preconfigured label to exclude particular namespaces or Pods: `log-shipper.deckhouse.io/exclude=true`.
It can help to stop collecting logs from a namespace or Pod without changing global configurations.
```
```